src/modules/queries/saga.test.ts

```diff
@@ -330,3 +335,152 @@ describe('deleteQuery()', () => {
     });
   });
 });
+
+describe('extractToEmail()', () => {
+  describe('Scenario 1: User successfully extracts to email', () => {
+    const it = sagaHelper(extractToEmailFlow());
```

Comment:
please use `const test = sagaHelper ...` to follow the pattern in other saga tests